src/fastga/models/performances/mission/mission_components/cruise.py

```diff
@@ -115,49 +115,39 @@ def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
         mass_t = mtow - (m_to + m_tk + m_ic + m_cl)
         atm = Atmosphere(cruise_altitude, altitude_in_feet=False)
         atm.true_airspeed = v_tas
-        mach = atm.mach
+
         previous_step = ()
+        self.flight_points = []
```

Comment:
Same remark as in climb.

Reply:
I can try to find something more beautiful maybe

Reply:
It's beautiful the way it is.